node/src/accountant/receivable_dao.rs

```diff
@@ -669,93 +700,62 @@ mod tests {
         assert_eq!(result, None);
     }
 
-    #[test]
-    fn receivables_fetches_all_receivable_accounts() {
-        let home_dir = ensure_node_home_directory_exists(
-            "receivable_dao",
-            "receivables_fetches_all_receivable_accounts",
-        );
-        let wallet1 = make_wallet("wallet1");
-        let wallet2 = make_wallet("wallet2");
-        let time_stub = SystemTime::now();
-        let subject = ReceivableDaoReal::new(
-            DbInitializerReal::default()
-                .initialize(&home_dir, true, MigratorConfig::test_default())
-                .unwrap(),
-        );
-
-        subject
-            .more_money_receivable(time_stub, &wallet1, 1234)
-            .unwrap();
-        subject
-            .more_money_receivable(time_stub, &wallet2, 2345)
-            .unwrap();
-
-        let accounts = subject
-            .receivables()
-            .into_iter()
-            .map(|r| ReceivableAccount {
-                last_received_timestamp: time_stub,
-                ..r
-            })
-            .collect::<Vec<ReceivableAccount>>();
-        assert_eq!(
-            accounts,
-            vec![
-                ReceivableAccount {
-                    wallet: wallet1,
-                    balance: 1234,
-                    last_received_timestamp: time_stub
-                },
-                ReceivableAccount {
-                    wallet: wallet2,
-                    balance: 2345,
-                    last_received_timestamp: time_stub
-                },
-            ]
-        )
+    fn make_connection_with_our_defined_sqlite_functions(
+        home_dir: &Path,
+    ) -> Box<dyn ConnectionWrapper> {
+        let init_config = DbInitializationConfig::test_default()
+            .add_special_conn_setup(BigIntDivider::register_deconstruct_for_sqlite_connection);
+        DbInitializerReal::default()
+            .initialize(home_dir, true, init_config)
+            .unwrap()
     }
 
     #[test]
     fn new_delinquencies_unit_slope() {
-        let pcs = PaymentThresholds {
+        fn wei_conversion(gwei: u64) -> i128 {
+            i128::try_from(gwei).unwrap() * WEIS_OF_GWEI
+        }
+        let payment_thresholds = PaymentThresholds {
             maturity_threshold_sec: 25,
             payment_grace_period_sec: 50,
             permanent_debt_allowed_gwei: 100,
             debt_threshold_gwei: 200,
             threshold_interval_sec: 100,
-            unban_below_gwei: 0, // doesn't matter for this test
+            unban_below_gwei: 0,
         };
         let now = now_time_t();
         let mut not_delinquent_inside_grace_period = make_receivable_account(1234, false);
-        not_delinquent_inside_grace_period.balance = pcs.debt_threshold_gwei + 1;
+        not_delinquent_inside_grace_period.balance_wei =
+            wei_conversion(payment_thresholds.debt_threshold_gwei + 1);
```

Comment:
I'm sure I've seen gwei_to_wei(), and I've also seen * WEIS_OF_GWEI...it seems to me that some or all of these could be combined into two globally-accessible conversion functions.